mods/ra/rules/structures.yaml

```diff
@@ -1141,28 +1128,13 @@ AFLD:
 		CameraActor: camera.paradrop
 		DisplayBeacon: true
 		BeaconPoster: pinficon
-	ParatroopersPower@armordrop:
-		OrderName: RussiaArmorDrop
-		Prerequisites: aircraft.russia
-		Icon: armordrop
-		ChargeTime: 450
-		Description: Armor Airdrop
-		LongDesc: Badgers drop a pair of\nHeavy Tanks anywhere on the map.
-		DropItems: 3TNK,3TNK
-		SquadSize: 2
-		SelectTargetSound: slcttgt1.aud
-		AllowImpassableCells: false
-		QuantizedFacings: 8
-		CameraActor: camera.paradrop
-		DisplayBeacon: true
-		BeaconPoster: armordropicon
 	AirstrikePower@parabombs:
 		OrderName: UkraineParabombs
 		Prerequisites: aircraft.ukraine
 		Icon: parabombs
-		ChargeTime: 270
+		ChargeTime: 360
 		Description: Parabombs
-		LongDesc: A Badger drops a load of parachuted\nbombs on your target.
+		LongDesc: A squad of Badgers drops parachuted\nbombs on your target.
```

Comment:
A squad generally refers to a group of people, while a squadron generally
refers to a group of aircraft. You wouldn't say that there was a school of
cows just because something else says there's a school of fish, would you?